.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/cpp
+{
+	"name": "C++",
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"settings": {
+        "cmake.configureArgs": [
+            "-DLIBCZI_BUILD_CURL_BASED_STREAM=ON",
+            "-DLIBCZI_BUILD_PREFER_EXTERNALPACKAGE_LIBCURL=ON",			
+            "-DLIBCZI_BUILD_DYNLIB=OFF",
+            "-DLIBCZI_BUILD_CZICMD=ON"
```

Comment:
We may want to consider having EIGEN3 on system rather than downloaded as part of build with `LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_EIGEN3` as well as `LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_ZSTD`

Reply:
with "zstd" I am a bit torn - or, I think we should strive to use "always the latest and greatest version of zstd", and this is ensured most easily by building it ourselves I reckon.

Reply:
I agree, we should try to keep zstd as up-to-date as possible with its releases

Reply:
added (and tested with) "using external eigen3"